config/config.go

```diff
@@ -266,6 +267,7 @@ type (
 		SuggestBlockWindow int    `yaml:"suggestBlockWindow"`
 		DefaultGas         uint64 `yaml:"defaultGas"`
 		Percentile         int    `yaml:"Percentile"`
+		ActionGasLimit     uint64 `yaml:"actionGasLimit"`
```

Comment:
It's not necessary to define another config field. We just need to pass `Genesis.ActionGasLimit` into api